zbench.zig

```diff
@@ -154,18 +154,27 @@ pub const Benchmark = struct {
         var p995_buffer: [128]u8 = undefined;
         const p995_str = try format.duration(p995_buffer[0..], percentiles.p995);
 
-        var avg_buffer: [128]u8 = undefined;
-        const avg_str = try format.duration(avg_buffer[0..], self.calculateAverage());
+        var avg_std_buffer: [128]u8 = undefined;
+        var avg_std_offset = (try format.duration(avg_std_buffer[0..], self.calculateAverage())).len;
+        avg_std_offset += (try std.fmt.bufPrint(avg_std_buffer[avg_std_offset..], " ± ", .{})).len;
+        avg_std_offset += (try format.duration(avg_std_buffer[avg_std_offset..], self.calculateStd())).len;
+        const avg_std_str = avg_std_buffer[0..avg_std_offset];
 
         var min_buffer: [128]u8 = undefined;
         const min_str = try format.duration(min_buffer[0..], self.minDuration);
 
         var max_buffer: [128]u8 = undefined;
         const max_str = try format.duration(max_buffer[0..], self.maxDuration);
 
-        std.debug.print("{s:<20} {s:<12} {s:<20} {s:<10} {s:<10} {s:<10}\n", .{ "benchmark", "time (avg)", "(min ... max)", "p75", "p99", "p995" });
-        std.debug.print("--------------------------------------------------------------------------------------\n", .{});
-        std.debug.print("{s:<20} \x1b[33m{s:<12}\x1b[0m (\x1b[94m{s}\x1b[0m ... \x1b[95m{s}\x1b[0m) \x1b[90m{s:<10}\x1b[0m \x1b[90m{s:<10}\x1b[0m \x1b[90m{s:<10}\x1b[0m\n", .{ self.name, avg_str, min_str, max_str, p75_str, p99_str, p995_str });
+        var min_max_buffer: [128]u8 = undefined;
+        const min_max_str = try std.fmt.bufPrint(min_max_buffer[0..], "({s} ... {s})", .{min_str, max_str});
+
+        // TODO: Very long benchmark-names can still cause ugly spacing. Although the solution is to simply
```

Comment:
TODO's without an issue, should be avoided. I created an [issue](https://github.com/hendriknielaender/zBench/issues/32) to track this. But thanks for mentioning it.